recipes/r-rserve/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
No, I mean this here: https://github.com/conda/conda-recipes/blob/master/r-packages/r-rserve/build.sh